homeassistant/components/connector/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "connector",
+  "name": "Connector",
+  "codeowners": ["@ConnectorGit"],
+  "config_flow": true,
+  "dependencies": ["network"],
+  "documentation": "https://www.home-assistant.io/integrations/connector",
+  "homekit": {},
```

Comment:
Should there be anything in this field? (If not, feel free to close this)

Reply:
Thanks for the reminder, I will delete this line